closure/goog/events/keyhandler.js

```diff
@@ -429,11 +429,11 @@ goog.events.KeyHandler.prototype.handleEvent = function(e) {
       }
     }
   } else if (
-      be.keyIdentifier &&
-      be.keyIdentifier in goog.events.KeyHandler.keyIdentifier_) {
+      be.key &&
```

Comment:
I'm not highly familiar with this code but this seems to cover an edge-case. Maybe the use of the deprecated `keyIdentifier` property is justified?
Please explain why you think that this is an issue.
Furthermore please add a test case for this specific edge-case and explain when this code will be reached.
Note that `keyIdentifier` is still being used in test code: https://github.com/google/closure-library/blob/8bbc453a8808aa77af91f959fb0dc97be058cd97/closure/goog/events/keyhandler_test.js#L705